src/scripts/clipperUI/panels/successPanel.tsx

```diff
@@ -9,12 +9,27 @@ import {ClipMode} from "../clipMode";
 import {ClipperStateProp} from "../clipperState";
 import {ComponentBase} from "../componentBase";
 import {Clipper} from "../frontEndGlobals";
+import {RatingsHelper, RatingsPromptStage} from "../ratingsHelper";
 import {Status} from "../status";
 
 import {SpriteAnimation} from "../components/spriteAnimation";
 
-class SuccessPanelClass extends ComponentBase<{ }, ClipperStateProp> {
-	onLaunchOneNoteButton() {
+import {DialogButton, DialogPanel} from "./dialogPanel";
+
+interface SuccessPanelState {
+	currentRatingsPromptStage?: RatingsPromptStage;
```

Comment:
I originally wanted a standalone RatingsPanel component too, but when I was thinking things through at the beginning, the MainController seemed like it would inherit too much ratings logic that it didn't need to know about in order to do so.

Now that the code is written, however, that result of that logic is available in the clipper state. So it's more obvious to me now that I can move that decision to the main controller #Pending